jans-cedarling/bindings/cedarling_python/cedarling_python.pyi

```diff
@@ -11,152 +11,106 @@ class BootstrapConfig:
-            - "jwt_sig_validation" (str): JWT signature validation status. Defaults to "enabled".
-            - "jwt_status_validation" (str): JWT status validation status. Defaults to "enabled".
-            - "jwt_signature_algorithms_supported" (list): Supported JWT signature algorithms. Defaults to an empty list.
-            - "at_iss_validation" (str): Access token issuer validation status. Defaults to "enabled".
-            - "at_jti_validation" (str): Access token JWT ID validation status. Defaults to "enabled".
-            - "at_nbf_validation" (str): Access token "not before" validation status. Defaults to "enabled".
-            - "at_exp_validation" (str): Access token expiration validation status. Defaults to "enabled".
-            - "idt_iss_validation" (str): ID token issuer validation status. Defaults to "enabled".
-            - "idt_sub_validation" (str): ID token subject validation status. Defaults to "enabled".
-            - "idt_exp_validation" (str): ID token expiration validation status. Defaults to "enabled".
-            - "idt_iat_validation" (str): ID token issued-at validation status. Defaults to "enabled".
-            - "idt_aud_validation" (str): ID token audience validation status. Defaults to "enabled".
-            - "userinfo_iss_validation" (str): User info issuer validation status. Defaults to "enabled".
-            - "userinfo_sub_validation" (str): User info subject validation status. Defaults to "enabled".
-            - "userinfo_aud_validation" (str): User info audience validation status. Defaults to "enabled".
-            - "userinfo_exp_validation" (str): User info expiration validation status. Defaults to "enabled".
-            - "id_token_trust_mode" (str): Trust mode for ID tokens. Defaults to "strict".
-            - "lock" (str): Lock mechanism status. Defaults to "disabled".
-            - "lock_master_configuration_uri" (str or None): Master configuration URI for locks. Defaults to None.
-            - "dynamic_configuration" (str): Dynamic configuration status. Defaults to "disabled".
-            - "lock_ssa_jwt" (str or None): JWT for SSA locks. Defaults to None.
-            - "audit_log_interval" (int): Interval for audit logs in seconds. Defaults to 0.
-            - "audit_health_interval" (int): Interval for health audit in seconds. Defaults to 0.
-            - "audit_health_telemetry_interval" (int): Interval for telemetry health audit in seconds. Defaults to 0.
-            - "listen_sse" (str): Server-Sent Events listening status. Defaults to "disabled".
-
-    Returns:
-        An instance of the class.
-
-    Raises:
-        KeyError: If any required configuration key is missing.
-        ValueError: If a provided value is invalid or extraction fails.
-    """
-
-    def disable_all_jwt_validation(self) -> None: ...
-
+    ) -> None:
+        """
+        Initialize a new instance of the class with configuration options.
```

Comment:
Yes, pyi file is what is loaded by Python editors and LSP, so parameters are essential to help developers.

Reply:
@SafinWasi 
Since we are going to use `dict` as config. How about in `pui` file add link to documentation, without describing each possible key of `dict`?  Updating documentation in many parts on changes is annoying and leads to mistypes.